website/docs/docs/authentication/legacy.md

```diff
@@ -0,0 +1,490 @@
+---
+title: Authentication
+sidebar_label: Authentication (Cognito)
+---
+
+:::info
+This article applies to accounts that use Cognito for authentication. This includes all accounts before Dec 11th 2023. For customers on SmartBear ID see [this guide](./main).
+:::
+
+To determine which authentication mode is enabled on your account, consult the following table:
+
+| SmartBear ID | Cognito | Both |
+|:---:|:--------------:|:--:|
+| <img border="1" width="250" src="/ui/login-sbid-only.png" description="Login Screen - SBID" /> | <img border="1" width="250" src="/ui/login-cognito-only.png" description="Login Screen - Cognito" /> | <img border="1" width="250" src="/ui/login-cognito-and-sbid.png" description="Login Screen - Both" />
+| If you only see this page when you visit your PactFlow Account, you are fully transitioned to SmartBear ID. See [this guide](./main).  | If you see a login page like this, you are on our Cognito authentication. | If there is a "SMARTBEAR ID" button on your login screen like this, you have _both_ enabled and are in the process of migrating to SmartBear ID. |
+
+To make it simpler to access PactFlow, we've rolled out Single Sign On (SSO) authentication with Github, Google and SAML2.0.
+
+## How it works
+
+SSO users are automatically provisioned when they first authenticate.
+
+**NOTE:** &nbsp; Changing these settings may require users to do a hard refresh in their browser (CTRL-F5 on Windows and Linux, CMD-R on Mac OSX) before the next time they try to login, or clear their browser cookies and cache.
+
+## Prerequisites
+
+If you are migrating to Single Sign On from username/password based authentication, you should consider enabling merging of identities based on email address, via the ["Consolidate User Logins by Email" system preference](/docs/user-interface/settings/preferences#consolidate-user-logins-by-email), otherwise you will end up with duplicate users.
```

Comment:
you should consider enabling the merging of identities based on an email address